changelog/unreleased/pull-246

```diff
@@ -0,0 +1,24 @@
+Enhancement: Enable socket activation by default and add network access restrictions in example systemd unit rest-server.service
```

Comment:
We usually don't add changelog entries for documentation updates. Either way, this changelog entry is far too long. So I'd suggest to remove it.

Reply:
Done